content/index.html.haml

```diff
@@ -45,9 +45,9 @@ homepage: true
 -# 2019 Board elections
 - slides << {:href => 'blog/2019/09/25/board-elections',
   :title => '2019 Jenkins Board and Officer elections',
-  :intro => "In November we will be electing three governing board members and two officers, namely Security and Events. Please register before Nov 17th and vote between Nov 11th and Nov 24th.",
-  :image => {:src => expand_link('/images/logos/needs-you/Jenkins_Needs_You-transparent.png'), :height => "300px"},
-  :call_to_action => {:text => 'More info', :href => expand_link('blog/2019/11/08/board-elections')}}
+  :intro => "In November 2019 we held the first Jenkins board elections elections ever. On behalf of the Jenkins community, we congratulate all elected board members and officers!",
+  :image => {:src => expand_link('/images/post-images/elections-2019/election-2019-opengraph.png'), :height => "300px"},
```

Comment:
This image does not look well due to the white background. May invent something better before the merge

Reply:
I did not, will probably do in a follow-up